test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -14,148 +14,126 @@ tags:
+   ```
+
+4. Modify the `rate-limit-alerts` to allow alerts to fire on a per minute basis:
+
+   ```sh
+   oc patch configmap rate-limit-alerts -n redhat-rhoam-operator -p '"data": {
+       "alerts": "{\n  \"api-usage-alert-level1\": {\n    \"ruleName\": \"RHOAMApiUsageLevel1ThresholdExceeded\",\n    \"level\": \"warning\",\n    \"minRate\": \"80%\",\n    \"maxRate\": \"90%\",\n    \"period\": \"1m\"\n  },\n  \"api-usage-alert-level2\": {\n    \"ruleName\": \"RHOAMApiUsageLevel2ThresholdExceeded\",\n    \"level\": \"warning\",\n    \"minRate\": \"90%\",\n    \"maxRate\": \"95%\",\n    \"period\": \"1m\"\n  },\n  \"api-usage-alert-level3\": {\n    \"ruleName\": \"RHOAMApiUsageLevel3ThresholdExceeded\",\n    \"level\": \"warning\",\n    \"minRate\": \"95%\",\n    \"period\": \"1m\"\n  }\n}"
+   }'
+   ```
+
+5. Patch the `rhoam` CR to specify BU, SRE and Customer email addresses:
+
+   NOTE: Replace `<rh_username>` references in the below commands with a valid Red Hat username. For example: `pamccart+BU@redhat.com`
+
+   Patch BU and SRE email addresses:
+
+   ```sh
+   oc patch rhmi rhoam -n redhat-rhoam-operator --type merge -p '{"spec":{"alertingEmailAddresses":{"businessUnit":"<rh_username>+BU@redhat.com", "cssre":"<rh_username>+SRE@redhat.com"}}}'
+   ```
+
+   Patch Customer email addresses with two emails to validate multiple addresses:
+
+   ```sh
+   oc patch rhmi rhoam -n redhat-rhoam-operator --type merge -p '{"spec":{"alertingEmailAddress":"<rh_username>+CUSTOMER1@redhat.com <rh_username>+CUSTOMER2@redhat.com"}}'
+   ```
+
+6. Verify email addresses are added to Alert Manager configurations
+
+   Open the Alert Manager console and login via Openshift:
+
+   ```sh
+   open "https://$(oc get route alertmanager-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
+   ```
+
+   Go to `Status -> Config` and check that the BU, SRE and Customer email addresses are included in the `BUAndCustomer` and `SRECustomerBU` receivers in the Alert Manager configuration where appropriate.
+
+7. From the Openshift console, retrieve the 3Scale admin password by going to `Secrets` > `system-seed` under the `redhat-rhoam-3scale` namespace and copying the `admin` password
+
+8. Next, open the 3Scale admin console and login with `admin` as the username and the retrieved password
+
+   ```sh
+   open "https://$(oc get route -n redhat-rhoam-3scale | grep 3scale-admin | awk {'print $2'})"
+   ```
+
+9. Click on `Ok, how does 3scale work?` and follow the 3Scale wizard to create an API
+10. Once on your API overview page, click on `Integration` on the left, then on `Configuration`
+11. Take note of the `example curl for testing` for `Staging-APIcast`
+12. Trigger API Usage alerts
+
+    | Alert to fire                        | `numRequests` | `interval` | `recipients`          |
+    | ------------------------------------ | ------------- | ---------- | --------------------- |
```

Comment:
Seein as all requests are now 200 can we move that into the command rather than having to paste it?

Reply:
Sure, can do 👍

Reply:
Table has changed again along with the `numrequests` based on latest changes. Can chat about it on Monday